internal/backup/collector_storage_stack_common.go

```
@@ -0,0 +1,335 @@
+	return uniqueSortedStrings(out)
+}
+
+func (c *Collector) safeCopySystemdMountUnitFiles(ctx context.Context) error {
+	if err := ctx.Err(); err != nil {
+		return err
+	}
+
+	base := c.systemPath("/etc/systemd/system")
+	info, err := os.Stat(base)
+	if err != nil {
+		return nil
```

golangci-lint (2.11.4)

[error] error is not nil (line 260) but it returns nil

(nilerr)